### PR TITLE
✨ Sync Presenters from People Profiles

### LINKED ISSUE #1080: ✨ Presenters - Source them from SSW.People
#### Description:
Currently, we are manually adding the content and profile images of presenters from [SSW.People](https://github.com/SSWConsulting/SSW.People), we need to find a way to make them a source of truth (E.g. a new field "blurb" on SSW.People.Profiles)
Ideally it should have the first 1 or 2 paragraphs

```[tasklist]
### Tasks
- [x] setup cron job to copy people profiles from ssw.people to local repo
- [x] profiles should be processed to reduce the amount of data being stored e.g. remove unnecessary fields, use first 1-2 paragraphs as profile
- [x] copy profile picture
- [x] People - create an API endpoint that contains people's info and returns a .json payload
```

### PR DESCRIPTION
**Pull request by tkapa**
- Affected routes: Presenters collection 

- Fixes #1080 

Done video link: https://youtu.be/Hsv80ibD7Vs

Adds a workflow that will get presenter data from SSW.People. 

Uses endpoints provided by SSW.People to get markdown data, and checks out SSW.People.Profile to get all of the appropriate images for Presenters without further downloads.

Creates a PR based on the changes, and enables auto-merge.

Before Merging:
- [x] SysAdmin/Repo Admin - add a GitHub Application with the appropriate Permissions to the Repo
- [x] SysAdmin/Repo Admin - Add secrets/`GH_APP_KEY` and variables/`GH_APP_ID` to the Repo
- [ ] ~~SysAdmin/Repo Admin - Enable the GH app to bypass the requirement for PR Approvals, and only require PR checks to pass~~

P.S. Don't worry about how many commits it took to get this right :)

### CODE REVIEW

**Comment by tkapa** [comment]
Hey @wicksipedia 
As per my email SSW.Website | GitHub App Transfer

I've had Warwick update the Repo with the GitHub App Secrets. So I think this should be ready to review.

File: .github/workflows/fetch-people-profiles.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+      - name: Checkout People Profiles
+        uses: actions/checkout@v4
+        with:
+          repository: "SSWConsulting/SSW.People.Profiles"
+          path: ${{ env.TEMPORARY_FOLDER}}
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
```
Comment (wicksipedia) [inline .github/workflows/fetch-people-profiles.yml]: can you use the version specified in the `.nvmrc`

**Review by wicksipedia — APPROVED** [review]

File: .github/workflows/fetch-people-profiles.yml (hunk 1 above)
Comment (tkapa) [reply .github/workflows/fetch-people-profiles.yml]: Done - I've updated the action to use `lts/*`

Comment (tkapa) [reply .github/workflows/fetch-people-profiles.yml]: Re-done - referencing `.npmrc`

**Comment by tkapa** [comment]
Closing - I'm going to merge this into a feature branch first.
New PR is here https://github.com/SSWConsulting/SSW.Website/pull/2916

**Pull request closed without merging**